laws-africa/indigo-kenyalaw
# Combine points in time into one document for compatibility layer

To test:

1. Import at least two points-in-time for a document, and ensure there are differences.
2. Download the `.legacyxml` version of the file.
3. Check for sections with `evolvingId`, which will appear multiple times.

<img width="713" alt="localhost:8000:api:v1:ke:act:2008:15:eng@2013-01-0" src="https://user-images.githubusercontent.com/4178542/54921903-a6059c00-4f0f-11e9-9982-bba9bd38d0aa.png">

<img width="657" alt="localhost:8000:api:v1:ke:act:2008:15:eng@2013-01-0" src="https://user-images.githubusercontent.com/4178542/54921883-97b78000-4f0f-11e9-94f2-2c88c582d340.png">
